spec/acceptance/instance_spec.rb

```diff
@@ -35,6 +35,13 @@ def get_instance(name)
         :volume_size => 8,
       }
 
+
+      # The value for this ENV var must be an existing IAM role in your Amazon account
+      # the IAM role must be available in the acceptance test region.
+      #
+      # When testing IAM roles you need to provide the IAM role name and the coresponding ARN.
+      @config[:iam_instance_profile_name] = ENV['IAM_ROLE_NAME'] if ENV['IAM_ROLE_NAME'] && ENV['IAM_ROLE_ARN']
```

Comment:
I think I'd probably prefer to test both. It's likely we'll do some consolidation of the acceptance tests at some point in the future, but I'd prefer to err on the side of better coverage for the moment.